og/bump/tagger.py

```diff
@@ -1,41 +1,91 @@
 import os
 import re
 import subprocess
+from logging import getLogger
 from operator import methodcaller
 from pathlib import Path
 
 import semver
 
-if __name__ == "__main__":
-    # Parse version file (Cannot import relative boo)
-    pkg_base = Path(os.getcwd())
-    ver_file = min([*pkg_base.glob('**/_version.py')], key = lambda path: len(path.resolve().parents))
-    ver_re = re.compile(r".*__version__ = [\"\'](v?)(.*)[\"\']")
-    leading_v, *[file_ver] = ver_re.match((ver_file).read_text().replace('\n', ' ')).groups()
 
+def main():
+    logger = getLogger(__name__)
```

Comment:
```suggestion
logger = getLogger(__name__)

def main():
```
this is how we're writing it everywhere else